contrib/openapi.yml

```diff
@@ -41,6 +41,50 @@ paths:
           description: OpenStack environment not found.
           schema:
             $ref: '#/definitions/Error'
+  /clusters/{id}/hosts:
+    get:
+      summary: List hosts in a cluster
+      parameters:
+        - name: id
+          type: string
+          pattern: *UUID_PATTERN
+          required: true
+          in: path
+          description: |
+            Unique identifier representing the specific cluster.
+      responses:
+        200:
+          description: List of all hosts in a cluster.
```

Comment:
what about changing
List of all hosts in a cluster
-->
List all hosts in the cluster.